src/config.h

```diff
@@ -8,6 +8,11 @@ struct Config {
 
   std::filesystem::path config_path;  // Path of the config directory
 
+  bool use_cuda_graphs{false};
+  // TODO: pass it from config file/generator ctor when serving stack is supported
+  // Hardcoded for now
+  size_t max_batch_size{16};
```

Comment:
I'm not quite sure how do we pass the max_batch into the config. One option is through config file from model builder, another option is to have an api in the generator. let me know if you have any preference

Reply:
it's from the user

Reply:
I feel it's better to be put into Generator